app/views/admin/protocols/_form.html.erb

```diff
@@ -1,9 +1,9 @@
 <div class="row">
   <div class="col-md-6">
-    <%= bootstrap_form_with(model: [:admin, protocol], local: true, label_errors: true) do |form| %>
-      <%= form.text_field :name, id: :protocol_name %>
-      <%= form.number_field :follow_up_days, id: :protocol_follow_up_days, help: "Note: Usually 28 days to ensure patients aren't reminded to return on days when facilities are closed." %>
```

Comment:
@rsanheim How can I make this field's title "Follow-up days"? I tried using a string, but I broke the form because it doesn't accept a value for the parameter "follow up days".

Reply:
Thanks @kpethtel, adding a `label: ` field fixed it :)